src/components/QuestionPage.js

```diff
@@ -49,18 +86,22 @@ class QuestionPage extends Component {
         this.setState({
           win: 1,
         });
+        this.props.earnedPointsAction(this.state.time * 5);
```

Comment:
Herhangi bir yerde birden fazla this.state veya this.props varsa destructive yapalim. 
cosnt { } = this.props

Reply:
Bu durum app de bir cok yerde vardi

Reply:
tamamdir hepsini duzelttim, this.state leri de const seklinde tanimladim